packages/jest-circus/src/legacy-code-todo-rewrite/jestAdapter.ts

```diff
@@ -77,7 +77,8 @@ const jestAdapter = async (
   });
 
   for (const path of config.setupFilesAfterEnv) {
-    const esm = runtime.unstable_shouldLoadAsEsm(path);
+    // TODO: remove ? in Jest 26
+    const esm = runtime.unstable_shouldLoadAsEsm?.(path);
 
     if (esm) {
       await runtime.unstable_importModule(path);
```

Comment:
for posterity, I didn't bother guarding this as `esm` will be falsy if the function doesn't exist